### DISCUSSION TITLE
Code mentors for Textual?

### DISCUSSION THREAD

**Discussion by evanpolster** [Textualize/textual#2907] [Q&A]
Quick question.

I am struggling a bit with learning Textual. I have a simple App (see attached) I created that is actually looking far better than I would have imagined, for only a handful of hours of effort thus far, but I still am getting peculiar layout problems. That, and the runtime type and object identification is throwing me for a bit of a loop.  (In days of old I  would have had this thing called a "variable associated with a class instance" to play with :-) 

For example, the layout of the app I have I would have thought would put the two containers right next to each other vertically yet the second one is apparently docked at the center or bottom when I requested dock:top. Just one example of many challenges I am running into.

Are there code mentors available for hire? 

Just curious,

_evan

![AwsMonitorApp](https://github.com/Textualize/textual/assets/5722134/f2380b95-7dce-4947-8c34-8bc26012c15a)

**Comment by davep** [comment]
You're very welcome to post questions here or over in the #help-wanted channel on our Discord server. Note that it's far easier to help diagnose the sorts of issues you mention above if it's accompanied by minimal-but-runnable code examples that illustrate what you're having trouble with. 

PS: you can still create widgets and assign references to to them to instance variables; it's all still Python. It's quite a common thing to do that.

**Reply by evanpolster** [reply]
Thanks for your reply!

I attempted to drag the .py and .css file into this discusssion board but it does not allow it.

I connected to Discord Textual discussion earlier today but it did not allow me to post. I apparently did not go to #help-wanted.

I will try there.

**Reply by davep** [reply]
The common methods of sharing code here would be to [quote the code in your post](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#quoting-code) (ideally if it's pretty short), use [a gist](https://docs.github.com/en/get-started/writing-on-github/editing-and-sharing-content-with-gists), or simply share a repository with the code inside.

**Comment by willmcgugan** [comment]
We're here to help as Dave says! You may also find this [how-to on building a layout](https://textual.textualize.io/how-to/design-a-layout/) to be helpful.

**Reply by evanpolster** [reply]
Ah. Okay. Outside to inside. Makes sense. Let me see if I can get this to work without bothering you.
